browser-test/src/support/ti_dashboard.ts

```diff
@@ -25,24 +25,32 @@ export class TIDashboard {
   }
 
   async createClient(client: ClientInformation) {
-    await this.page.fill('label:has-text("Email address")', client.emailAddress)
-    await this.page.fill('label:has-text("First name")', client.firstName)
-    await this.page.fill('label:has-text("Middle name")', client.middleName)
-    await this.page.fill('label:has-text("Last name")', client.lastName)
-    await this.page.fill('label:has-text("Date of birth")', client.dobDate)
-    await this.page.click('text="Add"')
+    await this.page.getByRole('button', {name: 'Add new client'}).click()
+    await waitForPageJsLoad(this.page)
+    await this.page.fill('#email-input', client.emailAddress)
+    await this.page.fill('#first-name-input', client.firstName)
+    await this.page.fill('#middle-name-input', client.middleName)
+    await this.page.fill('#last-name-input', client.lastName)
+    await this.page.fill('#date-of-birth-input', client.dobDate)
+    if (client.notes != undefined && client.phoneNumber != undefined) {
```

Comment:
suggestion: Should we check the notes and phone number independently? e.g.

```
if (client.notes != undefined) {
  await this.page.fill('#ti-note-input', client.notes)
}
if (client.phoneNumber != undefined) {
  await this.page.fill('#phone-number-input', client.phoneNumber)
}
```

Reply:
Sure!